app/assets/src/components/views/samples/SamplesView.jsx

```diff
@@ -199,6 +199,7 @@ class SamplesView extends React.Component {
         disabled
         icon={heatmapIcon}
         popupText="Heatmap"
```

Comment:
by the way, it would be nice to be consistent in how we phrase pop text as either verbs "Save a collection" or nouns "Heatmap", @happyimadesignr